src/execution/proposing.ts

```diff
@@ -1,13 +1,14 @@
 import { task, types } from "hardhat/config";
 import { multiSendLib, safeSingleton } from "../contracts";
-import { buildMultiSendSafeTx, buildSafeTransaction, calculateSafeTransactionHash, SafeTransaction, MetaTransaction } from "@gnosis.pm/safe-contracts";
+import { buildMultiSendSafeTx, buildSafeTransaction, calculateSafeTransactionHash, SafeTransaction, MetaTransaction, safeSignMessage } from "@gnosis.pm/safe-contracts";
 import { parseEther } from "@ethersproject/units";
 import { getAddress, isHexString } from "ethers/lib/utils";
-import { proposalFile, readFromCliCache, writeToCliCache, writeTxBuilderJson } from "./utils";
+import { calculateSafeTransactionHash2, proposalFile, readFromCliCache, writeToCliCache, writeTxBuilderJson } from "./utils";
```

Comment:
the 2 is because it's for V2? if so i would add the v2 as it allows to understand why we have 2 different versions.
If its the same version but calculates it differently maybe it's better other description or pass down a parameter and the function will choose which one to use

Reply:
No, it is actually an internal function of the SDK I need to expose. I will rename it properly. Thanks Pedro